tests/config/users.d/more_users.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<yandex>
+    <users>
+        <secondary>
```

Comment:
How about creating user via DDL?

Reply:
I forgot about that possibility :laughing: